pkg/reconciler/config.go

```diff
@@ -69,6 +77,27 @@ type JetStreamVersion struct {
 	StartCommand         string `json:"startCommand"`
 }
 
+func (g *GlobalConfig) GetStandardResources() corev1.ResourceRequirements {
```

Comment:
`GetDefaultContainerResources()`, and it always return a new instance, so that the invokers do not need to deepCopy.

Reply:
updated